cmd/rqlited/flags.go

```diff
@@ -206,6 +206,9 @@ type Config struct {
 
 	// MemProfile enables memory profiling.
 	MemProfile string
+
+	// OTLPDest sets the OTLP endpoint
+	OTLPDest string
```

Comment:
Just so we're clear, what did you set this to at launch time?

Reply:
```shellsession
$ cat << EOF > config.yml
receivers:
  otlp:
    protocols:
      grpc:
exporters:
  googlecloud:
    project: some-project-name
service:
  pipelines:
    traces:
      receivers: [otlp]
      exporters: [googlecloud]
EOF

$ otelcol-contrib \
  --config ./config.yml &

$ OTEL_SERVICE_NAME=rqlited.1 go run ./cmd/rqlited \
  --otlp-dest localhost:4317 \
  ~/node1 &

$ OTEL_SERVICE_NAME=rqlited.2 go run ./cmd/rqlited \
  --http-addr localhost:14001 \
  --raft-addr localhost:14002 \
  --otlp-dest localhost:4317 \
  --join localhost:4002 \
  ~/node2 &

$ OTEL_SERVICE_NAME=rqlite go run ./cmd/rqlite \
  --otlp-dest localhost:4317
```

Reply:
I should mention that this command-line argument is short and explicit, but doesn't expose the full feature set of the OTLP exporter. Before this PR is merged, I'd want to switch over to the [autoexport package](https://pkg.go.dev/go.opentelemetry.io/contrib/exporters/autoexport) which is configured using [these environment variables](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/).